GDJS/Runtime/events-tools/commontools.ts

```diff
@@ -458,6 +458,34 @@ namespace gdjs {
       export const trunc = function (x: float): number {
         return x | 0;
       };
+
+      /**
+       * Compute the X position when given an angle and distance relative to the starting position.
+       * This is also known as getting the cartesian coordinates of a 2D vector, using its polar coordinates.
+       * @param positionX The position on X-axis, in pixels.
+       * @param angle The angle, in degrees.
+       * @param distance The distance from the object, in pixels.
+       */
+      export const XFromAngleAndDistance = function (
```

Comment:
`getXFromAngleAndDistance` (expression names in GDevelop don't have get, because they are always getters, but function names in JS/C++ always start with a verb)